Civi/Api4/Query/SqlFunction.php

```diff
@@ -199,6 +199,13 @@ public static function getCategory(): string {
     return static::$category;
   }
 
+  /**
+   * @return array|null
```

Comment:
@colemanw can we beef up the comment block here - I find this class a bit tricky so if the comment block explained when it is called it would be helpful

Reply:
Ok @eileenmcnaughton I added to that plus some general explanation at the top of the file about what these functions actually do.